### PR TITLE
Feature: support FIXED render type for GridFLow

### PR DESCRIPTION
**Pull request by penguinolog**
* GridFlow knows the amount and size of contents + size of spacing
* Move GridFlow tests from test_container to test_grid_flow

##### Checklist
- [X] I've ensured that similar functionality has not already been implemented
- [X] I've ensured that similar functionality has not earlier been proposed and declined
- [X] I've branched off the `master` or `python-dual-support` branch
- [X] I've merged fresh upstream into my branch recently
- [X] I've ran `tox` successfully in local environment
- [X] I've included docstrings and/or documentation and/or examples for my code (if this is a new feature)

### CODE REVIEW

File: docs/manual/images/urwid_widgets_2.png
Comment (wardi) [inline docs/manual/images/urwid_widgets_2.png:1]: the fixed widget color is on the right in all the others, better to be consistent

**Review by wardi — APPROVED** [review]
other than the image color issue this looks good

File: docs/manual/images/urwid_widgets_2.png
Comment (penguinolog) [reply docs/manual/images/urwid_widgets_2.png:1]: I copy-pasted background from `urwid_widgets_``

Comment (wardi) [reply docs/manual/images/urwid_widgets_2.png:1]: interesting. I wonder why I flipped the order between urwid_widgets_1 and urwid_widgets_2 :thinking: 

Still, better to be consistent within the same diagram at least.

Comment (penguinolog) [reply docs/manual/images/urwid_widgets_2.png:1]: Done

**Pull request merged**